src/extensions/http_api/client.js

```diff
@@ -218,19 +218,25 @@ class HttpApiClient {
      * @param {string} [baseUrl] - The base url
      */
     makeRequest(method, path, baseUrl) {
```

Comment:
And here would be our unspecified URI :)
```suggestion
    makeRequest(method, uri, baseUrl) {
```

Reply:
yes, all done!